python/ray/actor.py

```diff
@@ -590,6 +593,10 @@ def _remote(self,
                 this actor or task and its children (see
                 :ref:`runtime-environments` for details).  This API is in beta
                 and may change before becoming stable.
+            max_pending_calls (int): Set the maximum number of pending calls
```

Comment:
It's worth mentioning that this pending calls is counted separately for each caller. 
Also, can you update the Java comment to be the same as this one?

Reply:
Java comment added.